src/admin/admin.service.ts

```diff
@@ -0,0 +1,162 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { v4 as uuid } from 'uuid';
+import { AdminStudentService } from '../student/admin-student.service';
+import { AuthService } from '../auth/auth.service';
+import { MailService } from '../mail/mail.service';
+import { HrService } from '../hr/hr.service';
+import { InsertStudentDto } from 'src/student/dto/insert-student.dto';
+import { CreateHrDto } from './dto/create-hr.dto';
+import { CreateHrResponse, InsertStudentResponse } from '../types';
+
+@Injectable()
+export class AdminService {
+    constructor(
+        @Inject(AdminStudentService) private adminStudentService: AdminStudentService,
+        @Inject(AuthService) private authService: AuthService,
+        @Inject(MailService) private mailService: MailService,
+        @Inject(HrService) private hrService: HrService,
+    ) {
+    }
+
+    async createUsersFromFile(jsonfile: any): Promise<InsertStudentResponse> {
```

Comment:
Dlaczego any?

Reply:
jak dawałem typ pliku json to nie mogłem na nim użyć buffera